File: python/Car.py
# jumlahKursi
# jumlahPintu

from Vehicle import Vehicle

class Car(Vehicle):
    __jumlahKursi = ""
    __jumlahPintu = ""

    def __init__(self, jumlahKursi = "", jumlahPintu = ""):
        self.__jumlahKursi = jumlahKursi
        self.__jumlahPintu = jumlahPintu

    # getter
        
    def getjumlahKursi(self):
        return self.__jumlahKursi
    
    def getjumlahPintu(self):
        return self.__jumlahPintu
    
    # setter
    
    def setjumlahKursi(self, jumlahKursi):
        self.__jumlahKursi = jumlahKursi

    def setjumlahPintu(self, jumlahPintu):
        self.__jumlahPintu = jumlahPintu

File: c++/Car.cpp
#pragma once
#include <iostream>
#include <string>
#include "Vehicle.cpp"
using namespace std;

class Car : public Vehicle {
private:
    string jumlahKursi;
    string jumlahPintu;

public:
    Car(string jumlahKursi = "", string jumlahPintu = "") {
        this->jumlahKursi = jumlahKursi;
        this->jumlahPintu = jumlahPintu;
    }

    string getJumlahKursi() {
        return jumlahKursi;
    }

    string getJumlahPintu() {
        return jumlahPintu;
    }

    void setJumlahKursi(string jumlahKursi) {
        this->jumlahKursi = jumlahKursi;
    }

    void setJumlahPintu(string jumlahPintu) {
        this->jumlahPintu = jumlahPintu;
    }
};
